docker/development/docker-compose-postgres.yml

```diff
@@ -359,7 +359,7 @@ services:
       DB_PASSWORD: ${AIRFLOW_DB_PASSWORD:-airflow_pass}
     entrypoint: /bin/bash
     command:
-      - "/opt/airflow/ingestion_dependency.sh"
+      - "source /opt/airflow/ingestion_dependency.sh"
```

Comment:
using source to run the script so that the script can export or set system environment variables. either environment variable exported in the script will be available only inside of the script or in the terminal that is used to execute the script.

Reply:
@pmbrull I`m not able to see any errors. i also try to look into GitHub action. But no error logs.